Makefile

```diff
@@ -77,8 +79,11 @@ tarball: $(PROMU)
 	@$(PROMU) tarball --prefix $(PREFIX) $(BIN_DIR)
 
 docker:
+ifeq ($(MACH), ppc64le)
+	$(eval DOCKERFILE=Dockerfile.ppc64le)
+endif
 	@echo ">> building docker image"
```

Comment:
Let's include the dockerfile we're using in this message. Something like `>> building docker image from $(DOCKERFILE)`.